packages/kbn-capture-oas-snapshot-cli/src/kibana_worker.ts

```diff
@@ -40,11 +49,21 @@ export type Result = 'ready';
     oss: false,
     runExamples: false,
     watch: false,
-  });
+  };
 
-  await root.preboot();
-  await root.setup();
-  await root.start();
+  if (serverless) {
+    // Satisfy spaces config for serverless:
+    set(settings, 'xpack.spaces.allowFeatureVisibility', false);
+    const { startKibana } = createTestServerlessInstances({
+      kibana: { settings, cliArgs },
+    });
+    const {} = await startKibana();
```

Comment:
NIT: just `await startKibana()`?

Reply:
suprised this passed linting! fixed in 38ee0239b8c